projects/dendrites/subnetworks/utils.py

```diff
@@ -0,0 +1,383 @@
+# flake8: noqa
```

Comment:
Please include the software license at the top of this file.

Reply:
Good catch